modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,158 @@ Module included in the following assemblies:
+
+[NOTE]
+====
+Integration with `cert-manager` and `istio-csr` is not supported on {ibmpowerProductName}, {ibmzProductName}, or {linuxoneProductName}.
+====
+
+=== Integration with external authorization systems 
+This release introduces a generally available method of integrating {SMProductName} with external authorization systems using the `action: CUSTOM` field of the `AuthorizationPolicy` resource. Use `envoyExtAuthzHttp` to delegate the access control to an external authorization system.
+
+=== Integration with external Prometheus installation
+
+This release introduces a generally available version of the Prometheus extension provider. You can expose metrics to the OpenShift Monitoring stack or a custom Prometheus installation by setting the `extensionProviders` field to `prometheus` in the `spec.meshConfig` specification. The telemetry object configures Istio proxies to collect traffic metrics. {SMProductShortName} only supports the Telemetry API for Prometheus metrics. 
+
+[source,yaml]
+----
+spec:
+  meshConfig:
+    extenstionProviders:
+    - name: prometheus
+      prometheus: {}
+---
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: enable-prometheus-metrics
+spec:
+  metrics:
+  - providers:
+    - name: prometheus
+----
+
+=== Single Stack IPv6 Support
+:FeatureName: Single Stack IPv6 Support
+This release introduces generally available support for Single Stack IPv6 clusters, providing access to a broader range of IP addresses. Dual Stack (IPv4/IPv6) clusters are not supported.
+
+[NOTE]
+====
+Single Stack IPv6 Support is not available on {ibmpowerProductName}, {ibmzProductName}, or {linuxoneProductName}.
+====
+
+=== {product-title} Gateway API Support
+:FeatureName: {product-title} Gateway API
+include::snippets/technology-preview.adoc[]
+This release introduces an updated Technology Preview version of {product-title} Gateway API Support. By default, the functionality is disabled.
+
+==== Enabling {product-title} Gateway API
+To enable the {product-title} Gateway API set the `enabled` field to `true` in the `techPreview.gatewayAPI` specification of the SMCP resource.
+
+[source,yaml]
+----
+spec:
+  techPreview:
+    gatewayAPI:
+      enabled: true
+----
+
+Previously, environment variables were used to enable the Gateway API.
+
+[source,yaml]
+----
+spec:
+  runtime:
+    components:
+      pilot:
+        container:
+          env:
+            PILOT_ENABLE_GATEWAY_API: "true"
+            PILOT_ENABLE_GATEWAY_API_STATUS: "true"
+            PILOT_ENABLE_GATEWAY_API_DEPLOYMENT_CONTROLLER: "true"
+----
+
+=== Infrastructure Node Deployment
+{SMProductShortName} control plane deployment is now supported and documented on OpenShift infrastructure nodes. For more information, see:
```

Comment:
In this case we're trying to say it's supported _and_ it's documented. In other words, the functionality worked but we never announced it because it was not documented. Now it's documented. This is how the PM/PO wanted to position the messaging.

Reply:
Done

Reply:
@rh-tokeefe I got your point. But, the phrase sounds a bit odd `is documented on OpenShift infrastructure nodes` conveys something is documented on nodes, which sounds incorrect. If i would say something `is documented on the portal or in the guide`, then also it would make sense. How we write or document about something on nodes?